### PR TITLE
Adding limit windows builds logic

### PR DESCRIPTION
**Pull request by atalman**
Adding limit windows builds logic
Cleaning up conda environment after validation has been run

### CODE REVIEW

File: .github/scripts/validate_binaries.sh [hunk 1]
```diff
@@ -41,5 +41,8 @@ else
         fi
 
         python  ./test/smoke_test/smoke_test.py
+
+        conda deactivate
```
Comment (weiwangmeta) [inline .github/scripts/validate_binaries.sh:45]: Should these be aligned with the else, i.e. directly above the fi?

**Review by weiwangmeta — COMMENTED** [review]
Please clarify the errors if the conda deactivate is not used and why those should be put in the else statement.

**Review by seemethere — APPROVED** [review]
LGTM, let's address @weiwangmeta's comments prior to merge

File: .github/scripts/validate_binaries.sh (hunk 1 above)
Comment (weiwangmeta) [reply .github/scripts/validate_binaries.sh:45]: Oh, nvm. It was only created inside this else statement. So it should be correct.

**Review by weiwangmeta — APPROVED** [review]
Please wait for green signal and merge.

**Pull request merged**